app/controllers/page_requests_controller.rb

```diff
@@ -37,7 +37,9 @@ def view_creators_rights
   end
 
   def login
-    if AUTH_CONFIG['shibboleth_enabled']
+    if current_user
+      redirect_to catalog_index_path
```

Comment:
I added a landing_page function when I was working on the welcome page:

https://github.com/uclibs/curate/blob/develop/app/controllers/concerns/curate_controller.rb#L41-L43

It would be good to use this instead of explicitly referencing the path, in case we decide to change it.